mlflow/odahuflow/trainer/helpers/conda.py

```diff
@@ -93,7 +91,7 @@ def update_model_conda_env(model_training: ModelTraining):
     """
 
     mlproject_file_path = _find_mlproject_file_path(model_training)
-    with open(mlproject_file_path) as f:
+    with open(mlproject_file_path, encoding='utf-8') as f:
```

Comment:
Actually, I don't understand why is it needed. 'utf8' encoding is default in python3 so it seems pretty redundant...

Reply:
It's the linter proposal. https://www.python.org/dev/peps/pep-0597/#id12
For me, it's easier to follow the recommendation than to think about what and where could happen.